### PR TITLE
589 julia introduction page

### LINKED ISSUE #589: Julia introduction page
https://hbrs-cse.github.io/Modellbildung-und-Simulation/12_matlab/01_erste_schritte_in_matlab.html

da wird unter MATLAB Alternativen ganz kurz Julia erwähnt, aber nur so „unter ferner liefen…“. Wenn man die Online-Inhalte singulär betrachtet, nicht so schlimm, aber es wäre sicher schöner, wenn man für Julia nun auch unter Online eine eigene Seite hätte, die das Buchkapitel 1.3 ergänzt.

### PR DESCRIPTION
**Pull request by PhiSpel**
# Description

closes #589 

## Checklist

 - [x] This pull request is associated to an issue
 - [ ] All used resources use a CC BY-SA 4.0 license or less and are marked appropriately, see our [contribution guide](https://github.com/hbrs-cse/Modellbildung-und-Simulation/blob/master/CONTRIBUTING.md).
 - [ ] All used resources have been added to [the one spreadsheet](https://docs.google.com/spreadsheets/d/1VzA6rIrLjV0GJee_7CQShjTJ-w86xugP6vsIzDmNSGs/edit#gid=763219261)
 - [ ] New exercises include learning requirements and learning goals.
 - [ ] Add someone else as reviewer and wait for approval before merging.

### CODE REVIEW

File: content/13_julia/00_julia.ipynb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "Im Gegensatz zu Matlab kann die frei verfügbare Sprache Julia in verschiedenen Entwicklungsumgebungen (auch engl. *integrated development environment*, kurz IDE) ausgeführt werden, enthält aber nicht automatisch alle Pakete, welche für das Bearbeiten der Aufgaben aus diesem Onlinebuch benötigt werden. Im Folgenden soll eine kurze Übersicht über die Installation der Sprache Julia, zwei mögliche IDEs und die Installation benötigter Pakete gegeben werden.\n",
+    "\n",
+    "## Installation der Programmiersprache Julia\n",
+    "\n",
+    "Die Installation der Sprache Julia selbst erfolgt am einfachsten über die Website der Entwickler (Stand Dezember 2023: https://julialang.org/downloads/). Dort kann der passende Installer für verschiedene Betriebssysteme ausgewählt werden (z. B. *64-bit-installer* des *current stable release* für Windows-64-bit-Systeme). Nachdem man diesen heruntergeladen und ausgeführt hat, ist die Sprache Julia und eine einfache Julia-Kommandozeile zum Ausführen von Julia-Befehlen (Letzteres als Programm bzw. App Julia, siehe Abbildung 1) verfügbar. Diese Kommandozeilenapplikation wird auch REPL (= read-evaluate-print-loop) genannt.\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:14]: Can you link the website (also the weblinks below)? Should be `[url](url)`

File: content/13_julia/00_julia.ipynb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "## Test der Installation\n",
+    "\n",
+    "Starten Sie die Julia-REPL über das Icon auf dem Desktop (Abbildung 1), im Startmenü, oder im Installationsordner und führen sie den Befehl **println(''Hello World'')** aus. Als Ausgabe sollte **Hello World** erscheinen. Julia ist somit erfolgreich und lauffähig installiert.\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:18]: Can you format code examples (also below) as code? "`" should do the trick, I think

File: content/13_julia/00_julia.ipynb [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "```{image} pic_Julia_REPL.png\n",
+    ":align: center\n",
+    "```\n",
+    "\n",
+    "<div style=\"text-align: right\"> Abbildung 1: Julia-REPL zur direkten Eingabe von Julia-Befehlen. </div> <br>"
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:24]: Did you create the image yourself?

File: content/13_julia/00_julia.ipynb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
...tiv kann eine neue Datei angelegt werden und Code geschrieben oder in diese hinein kopiert werden. Beim ersten Öffnen einer Julia-Datei nach dem Start der Entwicklungsumgebung wird jeweils einmal der *Julia Language Server* gestartet, erkennbar an einer Mitteilung in der unteren Leiste links. Ist der Start abgeschlossen, steht an gleicher Stelle eine Information zur aktuellen Julia-Version, z. B. *v1.8*. Vor dem Bearbeiten der Aufgaben aus diesem Onlinebuch müssen die dafür nötigen Pakete installiert werden. Dies kann entweder über die Julia-Kommandozeilen-App oder ein Julia-Terminal innerhalb von VS Code geschehen. Nach der Paketinstallation und beim Öffnen einer Julia-Datei indiziert VS Code die Pakete, erkennbar an einer Mitteilung in der unteren Leiste, links. Dies kann unter Umständen mehrere Minuten in Anspruch nehmen. Sind alle nötigen Pakete installiert und indiziert, kann eine geöffnetes Julia-Datei mit dem *Play*-Knopf oben rechts gestartet werden (alternativ auf dem Pfeil daneben *Julia: Execute active File in REPL* wählen). Text-Ausgabe erscheint während bzw. nach der Ausführung im Terminal am unteren Fensterrand, etwaige Plots werden in einem neuen Reiter geöffnet.\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:53]: Can you make this 2-4 paragraphs?

File: content/13_julia/00_julia.ipynb [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "## Julia in Jupyter Notebook\n",
+    "\n",
+    "Jupyter Notebook ist eine auf der Programmiersprache Python basierende Notebook-Umgebung, für die auch Julia-Unterstützung installiert werden kann. Eine einfache Anleitung zur Installation von Jupyter Notebook über Anaconda und die Einbindung von Julia findet sich unter: https://jupyter.org/try. Hinweis: Jupyter Notebook arbeitet mit **.ipynb**-Dateien und nicht mit **.jl**-Dateien. Am einfachsten ist die Verwendung, indem man Jupyter Notebook startet, unter New$\\to$Julia eine neue Datei mit Julia-Kernel erstellt und startet und den auszuführenden Code in diese einfügt. Falls im Code andere Dateien (z. B. ein selbstgeschriebener Löser in einer **.jl**-Dateien) eingebunden werden (erkennbar an **include**-Statements im Code), müssen diese Dateien sich im gleichen Verzeichnis befinden, in dem die aktuelle **.ipynb**-Datei ausgeführt wird. Alternativ kann der Inhalt der einzubindenden Dateien anstelle des **include**-Statements in den Hauptquellcode eingefügt werden. Mit dem Paket **Pluto.jl** bietet Julia auch eine eigene Alternative zu Jupyter Notebook.\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:57]: Can you make this 2-3 paragraphs?

File: content/13_julia/00_julia.ipynb [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    "- QuadGK\n",
+ 	  "- Dierckx.\n",
+    "\n",
+    "Für jedes der Pakete muss einmal obiger Befehl ausgeführt werden, also beispielsweise **import Pkg; Pkg.add(''DifferentialEquations'')** und die erfolgreiche Installation abgewartet werden. Nach Installation dieser Pakete sollte das Bearbeiten der Aufgaben in diesem Onlinebuch grundsätzlich in einer der unten genannten Entwicklungsumgebungen möglich sein (Je nachdem wie Sie programmieren besteht natürlich die Möglichkeit, dass Sie weitere Pakete benötigen).\n",
+    "\n",
+    "## Entwicklungsumgebungen\n",
+    "\n",
+    "Um komfortabel mit Julia-Dateien (Endung **.jl**) arbeiten zu können, benötigt man eine Entwicklungsumgebung. Zwei mögliche IDEs mit Julia-Unterstützung sind *Visual Studio Code* (eine klassische, aber kompakte IDE) oder *Jupyter Notebook* (insbesondere für alle, die sowieso schon eine Python- und/oder Anaconda-Installation auf ihrem Rechner haben).\n",
+    "\n",
+    "## Julia in Visual Studio Code\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:51]: Make this (and the next) subsections, using a third `#`.

File: content/13_julia/00_julia.ipynb [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    "\n",
+    "## Installation der Programmiersprache Julia\n",
+    "\n",
+    "Die Installation der Sprache Julia selbst erfolgt am einfachsten über die Website der Entwickler (Stand Dezember 2023: https://julialang.org/downloads/). Dort kann der passende Installer für verschiedene Betriebssysteme ausgewählt werden (z. B. *64-bit-installer* des *current stable release* für Windows-64-bit-Systeme). Nachdem man diesen heruntergeladen und ausgeführt hat, ist die Sprache Julia und eine einfache Julia-Kommandozeile zum Ausführen von Julia-Befehlen (Letzteres als Programm bzw. App Julia, siehe Abbildung 1) verfügbar. Diese Kommandozeilenapplikation wird auch REPL (= read-evaluate-print-loop) genannt.\n",
+    "\n",
+    "## Test der Installation\n",
```
Comment (PhiSpel) [inline content/13_julia/00_julia.ipynb:16]: Make this (and the next?) a subsection using a third `#`.

File: content/13_julia/00_julia.ipynb (hunk 3 above)
Comment (Florian314) [reply content/13_julia/00_julia.ipynb:24]: Ich nehme mal an das ist von gert, ich das aus unserem Buch genommen

Comment (Florian314) [reply content/13_julia/00_julia.ipynb:24]: Und das ist kleine Zitation

**Comment by PhiSpel** [comment]
Looks good to me now, I think you can merge

**Pull request merged**